docs/en/FAQ.md

```diff
@@ -50,6 +50,11 @@ interact with. Alluxio supports other language bindings including
 [Python]({{ '/en/api/FS-API.html' | relativize_url }}#python) and
 [Golang]({{ '/en/api/FS-API.html' | relativize_url }}#go).
 
+Alluxio can also be run as a [FUSE mount]({{ '/en/api/FUSE-API.html' | relativize_url }}). This enables
```

Comment:
shall we call this FUSE-API or POSIX API? I keep feeling FUSE is only implementation details

Reply:
Updated, thanks.